composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "JetBrains/phpstorm-stubs",
+    "description": "PHP runtime & extensions header files for PhpStorm",
+    "homepage": "http://jetbrains.com/phpstorm",
```

Comment:
Switch the homepage URL to https: https://www.jetbrains.com/phpstorm/  They even have an Extended Validation certificate that shows up the company name